.github/release_plan.md

```diff
@@ -56,11 +56,19 @@
     -   [ ] Create pull request against `release-YYYY.MM` (🤖)
     -   [ ] Merge pull request into `release-YYYY.MM`
 -   [ ] Make sure component governance is happy
+-   [ ] Turn off automatic uploads for insider builds from main. This prevents stable customers from getting insiders builds as they have the same engine version and higher build numbers.
 
 ## Release
 
 -   [ ] Publish the release
-    -   [ ] For manual
+    -   [ ] For an automated release
+        -   [ ] Create a commit which contains the words `publish` and `release` in it (you can use --allow-empty if needed)
+        -   [ ] Directly push (PR not required) the commit to the `release-xxxx.xx` branch
+        -   [ ] This commit will trigger the `release` stage to run after smoke tests. [Example run](https://github.com/microsoft/vscode-jupyter/actions/runs/702919634)
+        -   [ ] For release branches a mail will be sent to verify that the release should be published. Click the `Review pending deployments` button on the mail and deploy from the GitHub page. This will publish the release on the marketplace.
```

Comment:
Have we updated the list of people who get this mail?

Reply:
@joyceerhl we hadn't, but now we have. I've added ya for the next release.